GPU/Directx9/StateMappingDX9.cpp

```diff
@@ -99,7 +99,7 @@ void DrawEngineDX9::ApplyDrawState(int prim) {
 	if (!gstate.isModeClear()) {
 		textureCache_->ApplyTexture();
 
-		if (fboTexBindState_ = FBO_TEX_COPY_BIND_TEX) {
```

Comment:
this one is interesting! maybe have something to do with the wacky antialiasing slowdown..

Reply:
Yep, it would make sense.  Easy to miss...

-[Unknown]